File: CS/GridViewDetailColumnWidth/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;

namespace GridViewDetailColumnWidth
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			this.carsTableAdapter.Fill(this.carsDBDataSet.Cars);
			this.carSchedulingTableAdapter.Fill(this.carsDBDataSet.CarScheduling);
		}
	}
}

File: VB/GridViewDetailColumnWidth/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Grid

Namespace GridViewDetailColumnWidth
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Me.carsTableAdapter.Fill(Me.carsDBDataSet_Renamed.Cars)
			Me.carSchedulingTableAdapter.Fill(Me.carsDBDataSet_Renamed.CarScheduling)
		End Sub
	End Class
End Namespace
